{{cookiecutter.beat}}/tests/system/config/{{cookiecutter.beat}}.yml.j2

```diff
@@ -0,0 +1,83 @@
+################### Beat Configuration #########################
+
+
+
+############################# Output ##########################################
+
+# Configure what outputs to use when sending the data collected by the beat.
+# You can enable one or multiple outputs by setting enabled option to true.
+output:
+
+  ### File as output
+  file:
+    # Enabling file output
+    enabled: true
+
+    # Path to the directory where to save the generated files. The option is mandatory.
+    path: {{ output_file_path|default(beat.working_dir + "/output") }}
```

Comment:
My alternative would be that this file is generated similar to beatname.yml file. And the basic file has no "beat specific" things inside.